README.md

```diff
@@ -1,128 +1,51 @@
+Caikit streamlines the management of AI models for application usage by letting AI model authors focus on solving well known problems with novel technology. With a set of model implementations based on Caikit, you can:
 
-- Hosts AI models for inference and training so that models can be accessed through APIs
-- Provides access to the different training techniques as supported by a model
-- Supports models from different AI communities like HuggingFace ...
-- Allows applications to change model versions without any code changes
-- Compatability with other AI model runtimes
+- Run training jobs to create models from your data
+- Run model inference using data APIs that represent data as structures rather than tensors
+- Implement the right training techniques to fit the model, from static regexes to multi-GPU distribution
+- Merge models from diverse AI communities into a common API (e.g. `transformers`, `tensorflow`, `sklearn`, etc...)
+- Update applications to newer models for a given task without client-side changes
 
-## What Differentiates Caikit to Other AI Model Runtimes?
+## What Differentiates Caikit from Other AI Model Runtimes?
 
-Developers who write applications that consume AI models are not necessarily AI experts who understand the intricate details of the AI model that they use. Some would like to treat it as a "black box" where they give it input and it returns the output. This is similar in cloud computing whereby some users would like to deploy their applications to the cloud without detailed knowledge of the cloud infrastructure. The value for them is in their application and that is what is of most interest to them.
+Developers who write applications that consume AI models are not necessarily AI experts who understand the intricate details of the AI models that they use. Some would like to treat AI as a "black box function" where they give it input and it returns the output. This is similar in cloud computing whereby some users would like to deploy their applications to the cloud without detailed knowledge of the cloud infrastructure. The value for them is in their application and that is what is of most interest to them.
 
-Caikit provides an abstraction layer for application developers where they can consume an AI model through APIs independent of understanding the data form of the model. In other words, the input and output to the model is in a format which is easily programmable and does not require data transformations. This facilitates the model and the application to evolve independently of each other.
+Caikit provides an abstraction layer for application developers where they can consume a AI models through APIs independent of understanding the data form of the model. In other words, the input and output to the model is in a format which is easily programmable and does not require data transformations. This facilitates the model and the application to evolve independently of each other.
```

Comment:
find-replace typo? I think the original with singular references made sense here or we should pluralize all?
```suggestion
Caikit provides an abstraction layer for application developers where they can consume an AI model through APIs independent of understanding the data form of the model. In other words, the input and output to the model is in a format which is easily programmable and does not require data transformations. This facilitates the model and the application to evolve independently of each other.
```